model/yeast-GEM.txt

```diff
@@ -3937,12 +3937,12 @@ r_4575	H+[c] + 3-oxopropanoate[c] => acetaldehyde[c] + carbon dioxide[c] 		  0.0
 r_4576	NADP(+)[c] + (R)-2,3-dihydroxy-3-methylbutanoate[c] <=> H+[c] + NADPH[c] + 3-hydroxy-3-methyl-2-oxobutanoate[c] 		-1000.00	1000.00	  0.00
 r_4577	3-methyl-2-oxobutanoate[c] + H2O[c] => 2,3-dihydroxy-3-methylbutanoate[c] 		  0.00	1000.00	  0.00
 r_4578	H+[c] + NADPH[c] + 2-acetyllactic acid[c] => NADP(+)[c] + (R)-2,3-dihydroxy-3-methylbutanoate[c] 		  0.00	1000.00	  0.00
-r_4579	H+[c] + 2 pyruvate[c] => carbon dioxide[c] + (2S)-2-acetolactate[c] 		  0.00	1000.00	  0.00
-r_4580	NADP(+)[c] + (R)-2,3-dihydroxy-3-methylbutanoate[c] <=> H+[c] + NADPH[c] + (2S)-2-acetolactate[c] 		-1000.00	1000.00	  0.00
+r_4579	H+[c] + 2 pyruvate[c] => carbon dioxide[c] + (2S)-2-acetolactate[c] 	YMR108W	  0.00	1000.00	  0.00
+r_4580	NADP(+)[c] + (R)-2,3-dihydroxy-3-methylbutanoate[c] <=> H+[c] + NADPH[c] + (2S)-2-acetolactate[c] 	YLR355C	-1000.00	1000.00	  0.00
```

Comment:
@feiranl why were these reactions included in the first place? As part of metabolomics update #156, but is (2S)-2-acetolactate not the same as 2-acetyllactate? I wonder if this was missed when making that updating. As these two reactions seem to be replicates of `r_0096` and `r_0097` (in reverse), with the only difference is their compartment? But from the metabolomics data we do not know the compartment, so having it already present in the mitochondrion is sufficient? I'm not sure if this shouldn't actually be curated by removing duplicate reactions, instead of adding gene associations.

Reply:
@edkerk Yes. It would be better to remove the duplicated reactions.

Reply:
Due to (2S)-2-acetolactate/2-acetyllactic acid being practically the same metabolite, reactions `r_4576-r_4580` could be traced back to be duplicates (albeit in a different compartment, but their presence in the cytosol was only because metabolomics data cannot distinguish between compartments). So these could be removed, as specified in [`v8_6_0.m`](https://github.com/SysBioChalmers/yeast-GEM/blob/43d4d141df30e166246f7e0964cae4ef57bc3da0/code/modelCuration/v8_6_0.m#L63-L67).